client/src/ER/RingdownSection.js

```diff
@@ -9,9 +9,7 @@ import './RingdownSection.scss';
 function RingdownSection({ title, ringdowns, onStatusChange }) {
   const [isExpanded, setExpanded] = useState(true);
 
-  return ringdowns.length === 0 ? (
-    <></>
-  ) : (
+  return (
```

Comment:
It seemed really wierd to show an empty blank screen when no ringdowns are active, this change makes it so that you see Waiting (0), Incoming (0) headers at least...